sky/packages/sky/lib/src/widgets/editable_text.dart

```diff
@@ -129,7 +132,9 @@ class EditableString implements KeyboardClient {
     onUpdated();
   }
 
-  void submit(SubmitAction action) {}
+  void submit(SubmitAction action) {
+    onSubmitted();
```

Comment:
either assert onSubmitted is not null in the constructor, or null-check it here before using it, or add a comment explaining why it can't be null if you get here